.ci/setup_env_ubuntu.sh

```diff
@@ -12,7 +12,7 @@ source "/etc/os-release" || source "/usr/lib/os-release"
 source "${cidir}/lib.sh"
 
 echo "Update apt repositories"
```

Comment:
Thanks @GabyCT,  could you please comment this line to be consistent with the next commented line.